### DISCUSSION TITLE
Modify sligthly CifWriter to handle material structure data stacking in one file

### DISCUSSION THREAD

**Discussion by E-Veillon** [materialsproject/pymatgen#3397] [Ideas]
Could it be possible to change a bit the CifWriter.write_file() method to make it possible to use append file mode 'a' or 'at' ? Like this it would become possible for projects handling high throughtput treatment of cif data to write a list of structures concatenated in one CIF file.

**Comment by janosh** [comment]
https://github.com/materialsproject/pymatgen/pull/3399 will be available in the next release.
